basics/signals/package.json

```
@@ -45,13 +45,16 @@
   },
   "dependencies": {
     "@jupyterlab/application": "^3.0.0-rc.15",
+    "@jupyterlab/apputils": "^3.0.3",
     "@jupyterlab/launcher": "^3.0.0-rc.15",
     "@jupyterlab/mainmenu": "^3.0.0-rc.15",
     "@jupyterlab/translation": "^3.0.0-rc.15",
```

```suggestion
    "@jupyterlab/application": "^3.0.3",
    "@jupyterlab/apputils": "^3.0.3",
    "@jupyterlab/launcher": "^3.0.3",
    "@jupyterlab/mainmenu": "^3.0.3",
    "@jupyterlab/translation": "^3.0.3",
```

it is a good practice to be consistent on the jupyterlab dependencies.

---

Fixed as requested.